src/sentry/static/sentry/app/views/projectDashboard/index.jsx

```diff
@@ -93,79 +94,84 @@ const ProjectDashboard = createReactClass({
     const dateSince = this.getStatsPeriodBeginTimestamp(statsPeriod);
     const resolution = this.getStatsPeriodResolution(statsPeriod);
     const {orgId, projectId} = this.props.params;
+    const {name: orgName} = this.getOrganization();
+    const {name: projectName} = this.getProject();
```

Comment:
To @dcramer 's point you can extra `slug` here instead of the name which is no longer exposed in the ui.

Reply:
we could also start returning slug as name to make transitioning easier here

Reply:
Okay cool, I'll update this to use project slugs instead of names.